storybook/src/pages/public/ProductPage/ProductPage.tsx

```diff
@@ -16,15 +16,15 @@ import {
 export const ProductPage = () => (
   <>
     <Grid paddingTop="large">
-      <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+      <Grid.Cell span={{ narrow: 4, medium: 7, wide: 9 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
         <Breadcrumb>
           <Breadcrumb.Link href="#">Home</Breadcrumb.Link>
           <Breadcrumb.Link href="#">Stadspas</Breadcrumb.Link>
         </Breadcrumb>
       </Grid.Cell>
     </Grid>
     <Grid as="main" id="inhoud" paddingBottom="x-large">
-      <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+      <Grid.Cell span={{ narrow: 4, medium: 7, wide: 9 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
```

Comment:
The entire content is now as wide as a Content page header, I don't think that is correct.